src/constants.ts

```diff
@@ -82,10 +82,7 @@ export class AzureComponentsStorage {
   static readonly fileName = 'azureconfig.json';
 }
 
-export class GlobalConstants {
-  static readonly extensionId = 'vsciot-vscode.vscode-iot-workbench';
-  static readonly indentationSpace = 4;
-}
+export class GlobalConstants { static readonly indentationSpace = 4; }
```

Comment:
Throughout the code, the variable indentationSpace is only used when writing JSON content to file. Can we extract a utility method for writing content to JSON file so that indentationSpace is kept in that method, and will not be referenced everywhere throughout the code?

Reply:
Fixed.